### PR TITLE
Replace reference to quay image with origin image

### PR DESCRIPTION
**Pull request by bn222**

### CODE REVIEW

File: config/manifests/bases/sriov-network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ spec:
                 - name: NETWORK_RESOURCES_INJECTOR_IMAGE
                   value: quay.io/openshift/origin-sriov-dp-admission-controller:4.14
                 - name: OPERATOR_NAME
-                  value: sriov-network-operator
+                  value: quay.io/openshift/origin-sriov-network-operator:4.14
```
Comment (msherif1234) [inline config/manifests/bases/sriov-network-operator.clusterserviceversion.yaml:121]: this is the operator-name not the image ?

Comment (bn222) [reply config/manifests/bases/sriov-network-operator.clusterserviceversion.yaml]: thanks. fixed

**Review by sosiouxme — APPROVED** [review]
/lgtm

**Comment by wizhaoredhat** [comment]
@pliurh We need your input here. Kind of confused what this image is doing.

**Comment by dougbtv** [comment]
Looks like that reference to Peng's image got added (back?) last month in https://github.com/openshift/sriov-network-operator/commit/e0eae3ea8f015aeb2d46041d20c26fb399d952ea#diff-5e82d6924965456eb845bc9d26c26db233dada547bb8a04be884e486bda83f18R102

**Comment by dougbtv** [comment]
My guess is that it's a mistake and it should likely be for the `quay.io/openshift` namespace as Balazs changed it to here

**Comment by bn222** [comment]
It got added back because it was removed before it. The commit you are referring to @dougbtv is just a detail really. It has been part of the repo for longer.

**Pull request merged**